lib/matplotlib/backends/backend_qt5.py

```diff
@@ -820,7 +823,7 @@ def _export_values(self):
             QtGui.QFontMetrics(text.document().defaultFont())
             .size(0, text.toPlainText()).height() + 20)
         text.setMaximumSize(size)
-        dialog.exec_()
+        self._export_values_dialog.show()
```

Comment:
If we leave this as modal, will it block the main window, or just the subplot tool? I think it would be best if it blocked the tool, but not the main window, if possible.

Reply:
If I am not mistaken, everything happens on a single thread, so it would not be possible as exec_ would block the thread.